python/cudf/cudf/core/column/lists.py

```diff
@@ -21,6 +21,14 @@ def __init__(
             children=children,
         )
 
+    def __sizeof__(self):
+        n = 0
+        if self.nullable:
+            n += self.base_mask.size
+        for child in self.base_children:
+            n += child.__sizeof__()
+        return n
```

Comment:
It looks like we're getting the `base_mask` size and the `base_children` size instead of the `mask` size and the `children` size.

That being said, we should calculate it without using `.mask` or `.children` to avoid copy constructing new objects unnecessary.